### DISCUSSION TITLE
Indoor Cam C220 - Not working ?

### DISCUSSION THREAD

**Discussion by shkb42** [fuatakgun/eufy_security#1262] [General]
I recently integrated (& reintegrated couple of times) - All devices show up fine except my Indoor Cam C220 - It shows only debug options and nothing else. No control of modes, no feed, no properties at all. Tried to search to see if it is not supported, but couldn't find anything. Hence, starting the discussion to check if I am missing something ? 
![image](https://github.com/user-attachments/assets/fbb478bc-3e93-40b1-80ff-bbbabc9527b2)

**Comment by MartinsZB** [comment]
The same thing with Indoor Cam C210. I opened the Issue about it.
